code/__DEFINES/subsystems.dm

```diff
@@ -40,6 +40,7 @@
 // Subsystem init_order, from highest priority to lowest priority
 // The numbers just define the ordering, they are meaningless otherwise.
 
+#define INIT_ORDER_DBCORE 17
```

Comment:
I'm beginning to wonder if I should implement seperate ordering for shutdowns...